From 8ed87538176faad001baea4d36322f169cb08bf0 Mon Sep 17 00:00:00 2001
From: motten <motten@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 18 Jan 2017 12:32:33 +0000
Subject: [PATCH 0742/1000] MANTIS 0032431 \version 1.0.1.21 \li 17.01.2017 \li
 MOt - Mantis 0032431: Notification \li added the notification
 'CMD_MASSAIREBOOT_LOCAL'

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1143 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Interfaces/inc/Tools/Notification.h |  6 ++++--
 .../Massai/cpp/Tools/inc/local_def_msystools.h        | 11 ++++++++---
 2 files changed, 12 insertions(+), 5 deletions(-)

diff --git a/SelfServiceCommon/Interfaces/inc/Tools/Notification.h b/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
index c0b44402..428bdab7 100644
--- a/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
+++ b/SelfServiceCommon/Interfaces/inc/Tools/Notification.h
@@ -60,9 +60,11 @@ namespace MASSAI
       CMD_MASSAISTART = 4000,
       CMD_MASSAISTART_REMOTE, 
       CMD_MASSAISTART_SCHEDULED,
-        
+       
       CMD_MASSAISTOP_REMOTE, 
-      CMD_MASSAISTOP_SCHEDULED
+      CMD_MASSAISTOP_SCHEDULED,
+
+      CMD_MASSAIREBOOT_LOCAL
 
     } MCONTROL_MSG;
 
diff --git a/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h b/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
index f609d9d1..ed534ace 100644
--- a/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
+++ b/SelfServiceCommon/Massai/cpp/Tools/inc/local_def_msystools.h
@@ -1,5 +1,5 @@
-#define FILE_VER_BIN     1,0,1,20
-#define FILE_VER_STR     "FileVersion", "1.0.1.20"
+#define FILE_VER_BIN     1,0,1,21
+#define FILE_VER_STR     "FileVersion", "1.0.1.21"
 
 /*!
 \file
@@ -7,12 +7,17 @@
 
 \page massai_tools.lib Release Notes - massai_tools.lib
 
-@version 1.0.1.20
+@version 1.0.1.21
 
 \section changehistory Change history
 
 tools.lib
 
+\version 1.0.1.21
+\li 17.01.2017
+\li MOt - Mantis 0032431: Notification
+\li added the notification 'CMD_MASSAIREBOOT_LOCAL'
+
 \version 1.0.1.20
 \li 17.01.2017
 \li MOt - Mantis 0032431: ScheduledTask
-- 
2.41.0.windows.1

